frontend/components/layout/Sidebar.tsx

```diff
@@ -74,12 +74,14 @@ const Sidebar = () => {
                   {activeOrganisation?.name}{' '}
                 </span>
               </div>
-              <FaChevronDown
-                className={clsx(
-                  'transition ease opacity-0 group-hover:opacity-100 text-zinc-800 dark:text-zinc-100 flex-shrink-0 ml-2',
-                  open ? 'rotate-180 opacity-100' : 'rotate-0'
-                )}
-              />
+              {showOrgsMenu && (
+                <FaChevronDown
+                  className={clsx(
+                    'transition ease opacity-0 group-hover:opacity-100 text-zinc-800 dark:text-zinc-100 flex-shrink-0 ml-2',
+                    open ? 'rotate-180 opacity-100' : 'rotate-0'
+                  )}
+                />
+              )}
```

Comment:
This conditionally renders the chevron icon but still renders the entire button which remains clickable and shows an invisible empty menu. A better approach would be to show just a the org name and plan as a div rather than a button when there is no click action.

Create the Orglabel with the plan as a separate component:

```ts
           const OrgLabel = () => (<div className="p-2 text-neutral-500 cursor-pointer flex items-center justify-between w-full group bg-neutral-500/10 ring-1 ring-inset ring-neutral-400/10 rounded-lg"
            >
              <div className="flex flex-col gap-0.5 min-w-0">
                <div>
                  <PlanLabel plan={activeOrganisation?.plan!} />
                </div>
                <span className="truncate font-semibold tracking-wider text-lg">
                  {activeOrganisation?.name}{' '}
                </span>
              </div>
               {showOrgsMenu && (
                <FaChevronDown
                  className={clsx(
                    'transition ease opacity-0 group-hover:opacity-100 text-zinc-800 dark:text-zinc-100 flex-shrink-0 ml-2',
                    open ? 'rotate-180 opacity-100' : 'rotate-0'
                  )}
                />
              )}
            </div>)
```

When there are no orgs, you can do an early return with just the label, else show the menu component as before, using the OrgLabel component as a button instead:

```ts
if (!showOrgsMenu) return <OrgLabel />

return (
    <Menu as="div" className="relative inline-block text-left w-full">
        {({ open }) => (
          <>
            <Menu.Button><OrgLabel /></Menu.Button>
            <Transition....
```